include/boost/url/any_allocator.hpp

```diff
@@ -0,0 +1,95 @@
+//
+// Copyright (c) 2022 Alan Freitas (alandefreitas@gmail.com)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// Official repository: https://github.com/CPPAlliance/url
+//
+
+#ifndef BOOST_URL_ANY_ALLOCATOR_HPP
+#define BOOST_URL_ANY_ALLOCATOR_HPP
+
+#include <boost/url/detail/config.hpp>
+#include <boost/url/detail/any_allocator.hpp>
+
+#include <memory>
+#include <type_traits>
+
+namespace boost {
+namespace urls {
+
+/** A type-erased allocator with shared ownership.
+
+    This type satisfies the requirements for <em>Allocator</em>
+
+    @par Specification
+    @li <a href="https://en.cppreference.com/w/cpp/named_req/Allocator"
+       >Allocator (cppreference.com)</a>
+
+ */
+#ifdef BOOST_URL_DOCS
+template< class T >
+using any_allocator = __see_below __;
```

Comment:
There's an extra space that shouldn't be here, it should read `__see_below__`. The documentation is rendered incorrectly as a result: 
![image](https://user-images.githubusercontent.com/1503976/149649047-4e5ef5cc-1dd2-404a-a7b0-d322cfbec66a.png)